### PR TITLE
fix(chat): temporarily hide file unpublish (Issue #2632, #2180)

### PR DESCRIPTION
**Pull request by Gimir**
**Description:**

- temporarily remove files from unpublish wizard/request untill nested folder concept is ready

Issues:

- Issue #2632 #2180 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2649.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2649.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2649.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Publish/PublicationItemsList.tsx [hunk 1]
```diff
@@ -409,52 +414,51 @@ export const PublicationItemsList = memo(
...
-              dataQa="files-to-send-request"
-              className="!pl-0"
-            >
-              {files.length ? (
-                files.map((f) => (
-                  <div key={f.id} className="flex items-center gap-2">
-                    <FilesRow
-                      itemComponentClassNames={classNames(
-                        'w-full cursor-pointer truncate',
-                        publishAction === PublishActions.DELETE && 'text-error',
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Publish/PublicationItemsList.tsx:425]: it's temporary hiding and I don't think we need to remove this logic

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2649.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**